packages/code-du-travail-frontend/src/outils/indemniteLicenciement/components/AbsencePeriods.js

```diff
@@ -105,7 +105,7 @@ const CellHeader = styled.div`
   font-weight: 700;
 `;
 
-const DelButton = styled(Button).attrs(() => ({ type: "button" }))`
+const DelButton = styled(Button)`
```

Comment:
default type for button is submit, not button so we need to pass type="button" to DelButton

Reply:
ahh flute j'avais pas capté sry ! Je me demande si on ne devrait pas rajouter un type=button par défaut au component button et laisser l'utilisateur mettre type=submit quand il le souhaite ?

Reply:
Dans un premier temps, autant répliquer le principe de base et voir ensuite si on utilise plus l'un ou l'autre pour fixer la valeur par défaut qui nous intéresse